addons/product/models/product_pricelist.py

```diff
@@ -159,7 +159,7 @@ def _get_product_rule(self, product, *args, **kwargs):
         return self._compute_price_rule(product, *args, compute_price=False, **kwargs)[product.id][1]
 
     def _compute_price_rule(
-            self, products, quantity, currency=None, uom=None, date=False, compute_price=True,
+            self, products, quantity, *, currency=None, uom=None, date=False, compute_price=True,
             **kwargs
     ):
```

Comment:
IMO, if there is something to fix, it's the override, not the base method :thinking: